agent/src/main.rs

```diff
@@ -51,21 +51,21 @@ async fn run_agent() -> Result<()> {
     })?;
     let apiserver_client = K8SAPIServerClient::new(token_path.to_string());
 
-    // Get node and bottlerocketnode names
+    // Get node and BottlerocketShadow names
     let associated_node_name = env::var("MY_NODE_NAME").context(error::GetNodeName)?;
-    let associated_bottlerocketnode_name = format!("brn-{}", associated_node_name);
-
-    // Generate reflector to watch and cache BottlerocketNodes
-    let brns = Api::<BottlerocketNode>::namespaced(k8s_client.clone(), NAMESPACE);
-    let brn_lp = ListParams::default()
-        .fields(format!("metadata.name={}", associated_bottlerocketnode_name).as_str());
-    let brn_store = reflector::store::Writer::<BottlerocketNode>::default();
-    let brn_reader = brn_store.as_reader();
-    let brn_reflector = reflector::reflector(brn_store, watcher(brns, brn_lp));
-    let brn_drainer = try_flatten_touched(brn_reflector)
+    let associated_bottlerocketshadow_name = format!("brs-{}", associated_node_name);
```

Comment:
I don't love that we need to change a string to make this work. We should make this a function in `models::node`. Right now `BottlerocketShadowSelector::brs_resource_name()` is the closest thing we have, but to call that you would also need the UID, which we don't have here.

Maybe we can add a new method `models::node::brs_name_from_node_name()` and call that method from `BottlerocketNodeSelector::brs_resource_name()` as well.

Reply:
Sure, I'll make a method in `models::node` to get brs resource name, and call it on agent side. thanks!